Scripts/run-tests.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o pipefail
+exitCode=0
+
+function checkStatus {
+    if [[ $exitCode == 0 && $1 != 0 ]]; then
+        exitCode=$1
+    fi
+}
+
+function uploadCodecov {
+    bash <(curl -s https://codecov.io/bash)
+}
+
```

Comment:
@mkistler Ahhh, good point. I had thought about it, and had decided at the time to leave it out of the script. Travis tends to update to the latest version of Xcode soon after it is released, so the `$IOS_SDK` and `$IOS_DESTINATION` values need to be updated pretty often, or we tend to see some errors related to default iOS versions.

I forgot about running the script locally though! As I see it, we could take 3 approaches to solve this:
1. We could set default values if they have not been set by Travis already (via a conditional assignment, similar to this: `FOO=${VARIABLE:-default}`). The downside of this is that we would have to decide if we want to keep the values within the `travis.yml` in sync with the default values in the `run-tests.sh` script.
2. We could remove those variables from the `travis.yml` and just set them within the `run-tests.sh` script - which might be cleaner, actually, since the values aren't needed anymore within the `travis.yml` file at all. The only downside is that future maintainers of the repo will need to know that if there are Travis issues, they'll need to look at not only the `travis.yml` file but also the `run-tests.sh` script.
3. We could include a comment at the beginning of the `run-tests.sh` script specifying that if running the script locally, it's necessary to pass these two values when calling the script. So, running this via command line would look like: 
` IOS_SDK="iphonesimulator10.3" IOS10_DESTINATION="OS=10.3.1,name=iPhone 7" ./Scripts/run-tests.sh`

Haha...sorry about the long comment! Just wanted to express my thinking. I'm leaning towards approach 2 - I wanted to check in and see what you think before I push that change.